source/documentation/basic-project.md

```diff
@@ -81,6 +81,69 @@ generating the site.
 With this option passed, `{{ site.url }}/about` will now be generated as
 `http://my.dev.host/blog-skelton/output_dev/about` instead of `/about`.
 
+## CSS and JavaScript
+
+It is best practice to combine all your CSS files and all your JavaScript files
+into single files. That way they are simple to include in your templates and
+create less overhead for the browser loading the page. The recommended way to
+combine assets is with Symfony's
+[Webpack Encore](https://symfony.com/doc/current/frontend.html).
+
+The [Blog Skeleton](https://github.com/sculpin/sculpin-blog-skeleton) project
+uses Encore to manage assets. If you're setting up your project with Sculpin
+directly, not using the skeleton, install Encore using the
+[setup instructions](https://symfony.com/doc/current/frontend/encore/installation.html).
```

Comment:
i'd add one more paragraph here saying that you need to install yarn and link to https://yarnpkg.com/en/docs/install for installation instructions

Reply:
I'm not sure that's necessary, as the Encore instructions start off by telling you to install Node and Yarn, and link to the installation docs for both.

Reply:
The sculpin-blog-skeleton README mentions using yarn to install the JS dependencies, but doesn't talk about installing yarn itself. I think if we are going to mention that anywhere, it should be in that README. @dbu maybe you could add that to https://github.com/sculpin/sculpin-blog-skeleton/pull/61 ?

Reply:
:man_facepalming: Oh, you included that in https://github.com/sculpin/sculpin-blog-skeleton/pull/61 already, sorry. I still don't think it's necessary to note it separately here.

Reply:
k